test_conformance/extensions/cl_khr_command_buffer/command_buffer_test_base.h

```diff
@@ -129,6 +129,7 @@ class clCommandBufferWrapper {
     clCommandBufferWrapper &operator=(cl_command_buffer_khr rhs)
     {
         reset(rhs);
+        retain();
```

Comment:
Do we need this because we're using `clCommandBufferWrapper` for the queried object in the `InfoBuffer` test? I don't think querying the object should affect the reference count, so would prefer we used raw `cl_khr_command_buffer` objects to return the query value there.

Reply:
It's needed because we are referencing the same command buffer through `clCommandBufferWrapper`. When the wrapper is doing `clReleaseCommandBufferKHR` on the `cl_command_buffer_khr` object, there is a crash.